pydicer/analyse/data.py

```diff
@@ -44,53 +44,73 @@ class AnalyseData:
     Class that performs common analysis on converted data
 
     Args:
-        data_directory (str|pathlib.Path, optional): Directory in which data is stored. Defaults to
-            ".".
-        dataset (str, optional): The name of the dataset on which to run analysis. Defaults to
-            "nifti".
+        working_directory (str|pathlib.Path, optional): Directory in which data is stored. Defaults
+          to ".".
     """
 
-    def __init__(self, data_directory=".", dataset_name="data"):
-        self.working_directory = Path(data_directory)
-        self.dataset_directory = self.working_directory.joinpath(dataset_name)
+    def __init__(self, working_directory="."):
+        self.working_directory = Path(working_directory)
 
-    def get_all_computed_radiomics_for_dataset(self):
+    def get_all_computed_radiomics_for_dataset(self, dataset_name="data"):
         """Return a DataFrame of radiomics computed for this dataset
 
+        Args:
+            dataset_name (str, optional): The name of the dataset on which to run analysis.
+              Defaults to "data".
+
         Returns:
             pd.DataFrame: The DataFrame of all radiomics computed for dataset
         """
 
+        dataset_directory = self.working_directory.joinpath(dataset_name)
+
         dfs = []
-        for struct_dir in self.dataset_directory.glob("*/structures/*"):
-            for radiomics_file in struct_dir.glob("radiomics_*.csv"):
-                col_types = {
-                    "Contour": str,
-                    "Patient": str,
-                    "ImageHashedUID": str,
-                    "StructHashedUID": str,
-                }
-                df_rad = pd.read_csv(radiomics_file, index_col=0, dtype=col_types)
-                dfs.append(df_rad)
+        for converted_csv in dataset_directory.glob("*/converted.csv"):
```

Comment:
I guess the new block of code on line 103 is similar to this, perhaps we can make a function to reduce redundancy?